standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java

```diff
@@ -10330,6 +10330,108 @@ public boolean deletePartitionColumnStatistics(String catName, String dbName, St
     return ret;
   }
 
+  @Override
+  public boolean deletePartitionMultiColumnStatistics(String catName, String dbName, String tableName,
+                                                 String partName, List<String> partVals, List<String> colNames, String engine)
+          throws NoSuchObjectException, MetaException, InvalidObjectException, InvalidInputException {
+    boolean ret = false;
+    Query query = null;
+    dbName = org.apache.commons.lang3.StringUtils.defaultString(dbName,
+            Warehouse.DEFAULT_DATABASE_NAME);
+    catName = normalizeIdentifier(catName);
+    if (tableName == null) {
+      throw new InvalidInputException("Table name is null.");
+    }
+    try {
+      openTransaction();
+      MTable mTable = getMTable(catName, dbName, tableName);
+      MPartitionColumnStatistics mStatsObj;
+      List<MPartitionColumnStatistics> mStatsObjColl;
+      if (mTable == null) {
```

Comment:
Is this check `mTable == null` redundant? as we validate the `mPartition` afterwards

Reply:
I think we can keep this if condition. 
Because if mTable is null, the exception will be thrown from here, so we can know the "Table" does not exist. 
If the exception is thrown later with the mPartition, we can know the "Partition" does not exist.